plugin/src/main/resources/metadata/findbugs.xml

```diff
@@ -169,6 +170,7 @@
     <BugPattern type="OBJECT_DESERIALIZATION" abbrev="SECOBDES" category="SECURITY" cweid="502"/>
     <BugPattern type="DESERIALIZATION_GADGET" abbrev="SECDESGAD" category="SECURITY" cweid="502"  experimental="true"/>
     <BugPattern type="TRUST_BOUNDARY_VIOLATION" abbrev="SECTBV" category="SECURITY" cweid="501"/>
+    <BugPattern type="SCALA_SENSITIVE_DATA_EXPOSURE" abbrev="SECSDATALEAK" category="SECURITY" cweid="200"/>
```

Comment:
The abbreviation could be shorter. This is not intended to be readable. Could be "SECSDL". (Minor issue)

It's a good thing to add the Scala prefix. This is an idea that should be applied to Java and other APIs. (Comment)